core/new-gui/src/app/workspace/component/visualization-panel-content/visualization-panel-content.component.ts

```diff
@@ -0,0 +1,75 @@
+import { Component, Inject, OnInit, AfterViewInit } from '@angular/core';
+import { MAT_DIALOG_DATA } from '@angular/material/dialog';
+import * as c3 from 'c3';
+import { PrimitiveArray } from 'c3';
+
+interface DialogData {
+  table: object[];
+  chartType: c3.ChartType;
+}
+@Component({
```

Comment:
Add documentation to this componenet, see other componenets to check how the comments are written.  In general you need to write the following: 1. an overview of this component, 2. what does this component display and where is it on the UI? 3. short explanation of how it's implemented